mitodl/micromasters
# 5249: update image, quote, name for Eric Grimson

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/micromasters/issues/5249

#### What's this PR do?
Updates the: image, name, and quote used for the VP of Open Learning on the home page (http://mm.odl.local:8079/)

#### How should this be manually tested?

1. Grab this branch
2. Visit http://mm.odl.local:8079/ and verify the above updates are shown.

#### Screenshots (if appropriate)

![Screenshot 2023-02-09 at 8 46 26](https://user-images.githubusercontent.com/8311573/217830228-49b19905-4d93-48ff-ba8f-8986af7c03db.png)
![Screenshot 2023-02-09 at 8 46 35](https://user-images.githubusercontent.com/8311573/217830232-227fe63b-38e2-4591-b346-aaa79f25b27d.png)
